Kickstarter-iOS/Features/DashboardDeprecationBanner/DashboardDeprecationView.swift

```diff
@@ -0,0 +1,50 @@
+import Library
+import SwiftUI
+
```

Comment:
Also I know I needed `@available(iOS 15)` in my SwiftUI view, but you don't need it. My view needed it because of `@FocusState`.
Careful with adding that availability check because we do still support iOS 14.